OpenRA.Mods.Common/Effects/RallyPointIndicator.cs

```diff
@@ -22,11 +22,16 @@ class RallyPointIndicator : IEffect
 		readonly RallyPoint rp;
 		readonly Animation flag;
 		readonly Animation circles;
+		readonly IEnumerable<ExitInfo> exits;
 
-		public RallyPointIndicator(Actor building, RallyPoint rp)
+		CPos cachedLocation;
+		WPos[] targetLine;
+
+		public RallyPointIndicator(Actor building, RallyPoint rp, IEnumerable<ExitInfo> exits)
```

Comment:
Can you either accept an array of exits, or call `ToArray` in the constructor please?